src/proxy/cmsis-dap.ts

```diff
@@ -256,8 +256,8 @@ export class CmsisDAP extends EventEmitter implements Proxy {
      * @param sequence The sequence to send
      * @returns Promise
      */
-    public async swjSequence(sequence: BufferSource): Promise<void> {
-        const bitLength = sequence.byteLength * 8;
+    public async swjSequence(sequence: BufferSource, bitLength?: number): Promise<void> {
+        bitLength = bitLength || sequence.byteLength * 8;
```

Comment:
Minor (and subjective) suggestion. This could be written:
```suggestion
    public async swjSequence(sequence: BufferSource, bitLength: number = sequence.byteLength * 8): Promise<void> {
```

Reply:
Agreed, has the same effect and spares a line of code.